Mathlib/Topology/Category/CompHaus/EffectiveEpi.lean

```diff
@@ -35,6 +35,10 @@ As a consequence, we obtain instances that `CompHaus` is precoherent and preregu
 
 universe u
 
+attribute [local instance] CategoryTheory.ConcreteCategory.instFunLike
+/- Previously, this had accidentally been made a global instance, and we now turn it on locally as
+convenient. -/
```

Comment:
```suggestion
/-
Previously, this had accidentally been made a global instance,
and we now turn it on locally as convenient.
-/
attribute [local instance] CategoryTheory.ConcreteCategory.instFunLike
```
Comments always go *before* the thing they are commenting on! (Unless they are on the same line, I guess.)